lib/game_of_life_web/live/universe_live.ex

```diff
@@ -0,0 +1,34 @@
+defmodule GameOfLifeWeb.UniverseLive do
+  use Phoenix.LiveView
+  import Calendar.Strftime
+  alias GameOfLife.Universe
+
+  def render(assigns) do
+    #<h2 phx-click="boom">It's <%= strftime!(@date, "%r") %></h2>
+
+    ~L"""
+    <div>
+      <h1>It's time for... the Game... of... LIFE!</h1>
+      <p style="width: 300px;"><%= @cells %></p>
+    </div>
+    """
+  end
+
+  def mount(_session, socket) do
+    if connected?(socket), do: :timer.send_interval(1000, self(), :tick)
+    universe = Universe.Supervisor.start_child("u1", {8, 8})
+
+    {:ok, put_cells(socket)}
+  end
+
+  def handle_info(:tick, socket) do
+    # TODO get universe name from socket
+    Universe.tick("u1")
```

Comment:
This `tick` call returns the current state of the universe after all cells have been updated.  This should be assigned to the socket.  Then the `render` method can just do a loop rather than passing in a pre-rendered string